composites/Plugin/Shared/components/YoastInput.js

```diff
@@ -18,7 +18,11 @@ export const YoastInputLabel = styled.label`
 `;
 
 export const YoastInputField = styled.input`
-	border: 1px solid ${ colors.$color_input_border };
-	padding: 0.75em;
-	font-size: 1em;
+	&&& {
+		box-shadow: inset 0 1px 2px rgba(0,0,0,.07);
+		border: 1px solid ${ colors.$color_input_border };
+		padding: 0.75em;
+		font-size: 13px;
```

Comment:
I'd keep it to `1em` (which is equivalent to 13px in Gutenberg) unless there are specific reasons to use a value in pixels.